.travis.yml

```diff
@@ -6,26 +6,11 @@ language: python
 python:
   - "3.6"
 install:
-  - pip install 'pytest>=4.6' pytest-cov # testing 
-  - pip install 'sphinx<=2.4.4' --force-reinstall # python documentation tool
-  - pip install sphinx_bootstrap_theme # documentation theme
-  - pip install sphinx_automodapi # automated function documentation
-  - pip install nbsphinx # jupyter notebook documentation support
-  - pip install jupyter # jupyter notebooks
   - sudo apt-get install pandoc # pandoc for jupyter notebooks
   - sudo apt install python-pydot python-pydot-ng graphviz # graphviz for class inheritance diagrams in docs
```

Comment:
@JonathanWenger is there a reason to install python-pydot and python-pydot-ng via apt, and not via pip? It would seem more in line with the other python packages

Reply:
Well, I guess they both closely belong to graphviz, so I guess even though this way it introduces a second way to install python packages, there is still a valid reason to group them together.

Reply:
No reason. I copied this from a sphinx buide on how to display inheritance graphs.